org.eclipse.xtext.builder/model/BuilderState.genmodel

```diff
@@ -4,7 +4,7 @@
     editorDirectory="" modelPluginID="org.eclipse.xtext.builder" modelName="BuilderState"
     rootExtendsClass="org.eclipse.emf.ecore.impl.MinimalEObjectImpl.Container" testsDirectory=""
     importerID="org.eclipse.emf.importer.ecore" complianceLevel="8.0" copyrightFields="false"
-    runtimeVersion="2.20">
+    runtimeVersion="2.29">
```

Comment:
@szarnekow when i regenerate, i am getting compile errors

Reply:
I'd assume the `@Overrides` have been removed manually before.